### ISSUE TITLE
Solo dex buy, sell exit the app

### ISSUE THREAD

**Issue by JernejaSen** [sologenic/deprecated-sologenic-expo-wallet#661]
Hey, I have a problem wirh SOLO DEX wallet since Sunday. I wrote to support but nothing happens. When I want to sell or buy, it exit the app.

**Comment by cserhati69** [comment]
Me too, no sell no buy what a fck???

**Comment by cserhati69** [comment]
I got respons from Solo Dex
They working on the problem and will be announcement very soon on the web app

**Comment by Jameslee1029** [comment]
I have a same problem. I lost a lot of money!!! Dam

**Comment by JernejaSen** [comment]
They said the same to me. But is frustrating

**Comment by babszz333** [comment]
Yes it is frustrating that the sell or buy buttons are frozen. Thank you for reporting this issue. Hopefully we do not miss the bull run
